code/datums/status_effects/drug_effects.dm

```diff
@@ -58,3 +58,39 @@
 	name = "Seizure"
 	desc = "FJOIWEHUWQEFGYUWDGHUIWHUIDWEHUIFDUWGYSXQHUIODSDBNJKVBNKDML <--- this is you right now"
 	icon_state = "paralysis"
+
+/datum/status_effect/stoned
+	id = "stoned"
+	duration = 10 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/stoned
+	status_type = STATUS_EFFECT_REFRESH
+	var/original_eye_color
+
+/datum/status_effect/stoned/on_apply()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		original_eye_color = human_owner.eye_color
+		human_owner.add_movespeed_modifier(/datum/movespeed_modifier/reagent/cannabis) //couchlock
+		human_owner.eye_color = BLOODCULT_EYE //gives cultists plausible deniability
+		human_owner.update_body() //updates eye color
+		ADD_TRAIT(human_owner, BLOODSHOT_EYES, type) //dilates blood vessels in eyes
+		ADD_TRAIT(human_owner, TRAIT_CLUMSY, type) //impairs motor coordination
+		SEND_SIGNAL(human_owner, COMSIG_ADD_MOOD_EVENT, "stoned", /datum/mood_event/stoned) //+15 High on Smokeleaf
+		human_owner.sound_environment_override = SOUND_ENVIRONMENT_DRUGGED
+	return TRUE
+
+/datum/status_effect/stoned/on_remove()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		human_owner.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/cannabis)
+		human_owner.eye_color = original_eye_color
+		human_owner.update_body()
+		REMOVE_TRAIT(human_owner, BLOODSHOT_EYES, type)
+		REMOVE_TRAIT(human_owner, TRAIT_CLUMSY, type)
+		SEND_SIGNAL(human_owner, COMSIG_CLEAR_MOOD_EVENT, "stoned")
+		human_owner.sound_environment_override = SOUND_ENVIRONMENT_NONE
+
+/atom/movable/screen/alert/status_effect/stoned
+	name = "Stoned"
+	desc = "dood u high as shit"
```

Comment:
literally how can anyone be against this pr

Reply:
FJOIWEHUWQEFGYUWDGHUIWHUIDWEHUIFDUWGYSXQHUIODSDBNJKVBNKDML <--- this is you right now

Reply:
That was the precedent that made think this would be okay also it is perfectly in tone with being high which should be kind of meta tbh